.github/workflows/push.yml

```diff
@@ -24,6 +24,7 @@ env:
   DB_PRIVATEKEY: ${{ secrets.SNOWFLAKE_DB_PRIVATEKEY }}
   DB_DATABASE: snowflake_writer
   DB_WAREHOUSE: snowflake_writer
+  DB_ROLE_NAME: snowflake_writer
```

Comment:
Tohle je ale "default role name" takze to neni uplne idealni, ale asi staci

Reply:
jo je, ale jde hlavně o to aby se to poslalo v connection stringu :)